packages/block-editor/src/components/list-view/index.js

```diff
@@ -89,11 +98,64 @@ function ListView(
 		[ draggedClientIds ]
 	);
 	const selectEditorBlock = useCallback(
```

Comment:
This callback does a lot more than selecting an editor block now so maybe its name should change to `updateBlockSelection` or something like that.

Reply:
Oh, nice idea. Happy to update it 👍